cmd/vcert/commands.go

```diff
@@ -165,6 +165,15 @@ var (
 		Usage:     "To enroll a SSH Certificate",
 		UsageText: `vcert sshenroll -u https://tpp.example.com -t <TPP access token> --template <val> --id <val> --principal bob --principal alice --valid-hours 1`,
 	}
+
+	commandSshGetConfig = &cli.Command{
+		Before:    runBeforeCommand,
+		Name:      commandSshGetConfigName,
+		Flags:     sshGetConfigFlags,
+		Action:    doCommandSshGetConfig,
+		Usage:     "To get the CA public key and principals",
```

Comment:
In the SSH Protect UI, we use Default Principals. Maybe, it makes sense to be consistent in the terminology.

```suggestion
		Usage:     "To get the SSH CA public key and default principals",
```

Reply:
fixed on the most recent commits